sunpy/net/tests/test_remote_urls.py

```diff
@@ -0,0 +1,74 @@
+import json
+import socket
+from os.path import isfile
+from urllib.error import HTTPError
+from urllib.request import urlopen
+
+import pytest
+import requests
+
+
+def pytest_generate_tests(metafunc):
+    funcarglist = [None]
+    if isfile(metafunc.config.getini("intercept_dump_file")):
```

Comment:
I am not sure I see the benefits to dumping this info to a file over the terminal?

Reply:
not sure about what are you referring, dumping urls to a file??

Reply:
Yeah.

Reply:
I feel like dumping is a better option rather than intercepting each time the user requests `remote_status`; added benefits is status during `remote_data` run without invoking a intercept session.
The other way I see is to intercept + ping in the plugin itself, which I think is a little tricky to do.

Reply:
I'm a little conflicted now; dumping the urls is good as an option to the user not default.

I feel like there is a cross-road ahead:
- Plugin calls another instance of pytest at the end and ping the urls there; (due to the monkeypatch, the urls cannot be pinged during the initial run)
- set the dump file name to something like ``.intercept`` and ping them inside sunpy online tests.

What's your take on this??

Reply:
Leave it, if it lives in the plugin externally, I have no problems with it.